nixos/doc/manual/development/option-types.section.md

```diff
@@ -197,13 +197,15 @@ merging is handled.
 
 Submodules are detailed in [Submodule](#section-option-types-submodule).
 
-`types.submodule` *`o`*
+`types.submodule` *`module`*
 
-:   A set of sub options *`o`*. *`o`* can be an attribute set, a function
-    returning an attribute set, or a path to a file containing such a
-    value. Submodules are used in composed types to create modular
-    options. This is equivalent to
+    This type is equivalent to
```

Comment:
I think this sentence should come after the special `:` syntax that seems to mark the start of the definition, but also perhaps also move it one sentence later? Between here:

> to `imports`). Submodules can